Source/WebCore/dom/TrustedTypePolicyFactory.idl

```diff
@@ -0,0 +1,23 @@
+[
+    EnabledBySetting=TrustedTypesEnabled,
+    Exposed=(Window,Worker)
+]
+interface TrustedTypePolicyFactory {
+    TrustedTypePolicy createPolicy(
+        DOMString policyName, optional TrustedTypePolicyOptions policyOptions = {});
+    boolean isHTML(any value);
+    boolean isScript(any value);
+    boolean isScriptURL(any value);
+    readonly attribute TrustedHTML emptyHTML;
+    readonly attribute TrustedScript emptyScript;
+    DOMString? getAttributeType(
+        DOMString tagName,
+        DOMString attribute,
+        optional DOMString elementNs = "",
+        optional DOMString attrNs = "");
+    DOMString? getPropertyType(
+        DOMString tagName,
+        DOMString property,
+        optional DOMString elementNs = "");
+    readonly attribute TrustedTypePolicy? defaultPolicy;
```

Comment:
nit: Can you add a blank line before / after each multi-line item? Makes it easier to read.